semesterly/settings.py

```diff
@@ -87,10 +87,10 @@ def get_secret(key):
 SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = get_secret('SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET')
 SOCIAL_AUTH_FACEBOOK_KEY = get_secret('SOCIAL_AUTH_FACEBOOK_KEY')
 SOCIAL_AUTH_FACEBOOK_SECRET = get_secret('SOCIAL_AUTH_FACEBOOK_SECRET')
-SOCIAL_AUTH_AZUREAD_TENANT_OAUTH2_KEY = get_secret('SOCIAL_AUTH_AZURE_TENANT_KEY')
-SOCIAL_AUTH_AZUREAD_TENANT_OAUTH2_SECRET = get_secret('SOCIAL_AUTH_AZURE_TENANT_SECRET')
-SOCIAL_AUTH_AZUREAD_TENANT_OAUTH2_TENANT_ID = get_secret('SOCIAL_AUTH_AZURE_TENANT_ID')
-JWT_AUTH_SECRET = get_secret('JWT_AUTH_SECRET')
+SOCIAL_AUTH_AZURE_TENANT_KEY = get_secret('SOCIAL_AUTH_AZURE_TENANT_KEY')
+SOCIAL_AUTH_AZURE_TENANT_SECRET = get_secret('SOCIAL_AUTH_AZURE_TENANT_SECRET')
+SOCIAL_AUTH_AZURE_TENANT_ID = get_secret('SOCIAL_AUTH_AZURE_TENANT_ID')
```

Comment:
The left side of the three above arguments should not have been changed; those names belong to the social_auth library

Reply:
Resolved in next commit